lib/modules/hosts/pornbot.js

```diff
@@ -0,0 +1,89 @@
+//Borrowed heavily from gfycat.js
+modules['showImages'].siteModules['pornbot'] = {
+	domains: [ 'pornbot.net' ],
+	detect: function(href, elem) {
+		return (/^https?:\/\/(?:w{3}\.)?(?:v\.)?pornbot\.net\/[a-z0-9]{8,}$/i).test(href);
+	},
+	handleLink: function(elem) {
+		var hashRe = /^https?:\/\/(?:w{3}\.)?(?:v\.)?pornbot\.net\/[a-z0-9]{8,}$/i,
```

Comment:
Needs a capturing group to extract the video ID:

    var hashRe = /^https?:\/\/(?:w{3}\.)?(?:v\.)?pornbot\.net\/([a-z0-9]{8,})$/i,